src/core/analysis/mcmc/Mcmcmc.cpp

```diff
@@ -1312,7 +1312,7 @@ void Mcmcmc::synchronizeValues( bool likelihood_only )
         {
             if ( pid == pid_per_chain[i] )
             {
-                MPI_Send(&results[i], 1, MPI_DOUBLE, (int)active_PID, 0, MPI_COMM_WORLD);
+                MPI_Send(&results[i], 1, MPI_DOUBLE, active_PID, 0, MPI_COMM_WORLD);
```

Comment:
what type was `active_PID` before it was converted to `int`?

Reply:
It is of type `size_t`.

Reply:
Hmm.... let's keep an eye on that...
If it keeps happening, we could revert this change and then re-introduce pieces of it.